src/browserview/BrowserComponent.java

```diff
@@ -235,6 +265,34 @@ public void showIssue(int id) {
 	 * Takes care of running it on a separate thread, and normalises error-handling across
 	 * all types of code.
 	 */
+	private boolean isBrowserActive(){
+		try {
+			if (driver.getCurrentUrl().isEmpty() && driver != null){
+				return false;
+			}
+			else if (driver == null){
+				return false;
+			}
+			return true;
+		}
+			catch (WebDriverException e) {
+				switch (BrowserComponentError.fromErrorMessage(e.getMessage())) {
+				case NoSuchWindow:
+					logger.info("Chrome was closed.");
+				default:
+					break;
+				}
+				return false;
+			}
+	}
+	
+	private void resetBrowser(){
+		logger.info("Chrome not responding.");
+		quit();
```

Comment:
`quit` was meant to be called only once, on application exit. Since `driver.quit()` kills the process, I guess it's no longer needed.